volume/mocks/Driver.go

```diff
@@ -0,0 +1,70 @@
+package mocks
```

Comment:
you need an extra linebreak after the boilerplate because godeps will assume that this huge comment applies to the package